src/colmap/image/warp.cc

```diff
@@ -57,9 +57,11 @@ void WarpImageBetweenCameras(const Camera& source_camera,
                              const Camera& target_camera,
                              const Bitmap& source_image,
                              Bitmap* target_image) {
-  CHECK_EQ(source_camera.width, source_image.Width());
-  CHECK_EQ(source_camera.height, source_image.Height());
-  CHECK_NOTNULL(target_image);
+  THROW_CHECK_EQ(source_camera.width,
+                 static_cast<size_t>(source_image.Width()));
```

Comment:
@ahojnnes  I had to fix quite a lot of these annoying warnings. Any reason why `Bitmap::{Width,Height}` return `int` instead of `size_t` ?

Reply:
No, not really. Can be changed.